Artifacts.toml

```diff
@@ -8,8 +8,8 @@ lazy = true
 os = "macos"
 
     [[CUDA.download]]
-    sha256 = "7164d0f49f394464e280e2262d5f0e27a6433c6d5bb3c2a340bc1c8402054010"
```

Comment:
This PR changed URLs and sha256, but not the git-tree-sha1s, which is probably the reason of the discrepancy in https://github.com/JuliaGPU/CUDA.jl/pull/1182/files#r780513544

Reply:
No, this is legit. [JuliaBinaryWrappers/CUDA_loader_jll.jl@ecb51de#diff-1de81b882b140a45cfbcedd944f143a37759b544a9efc0f7008302578afa6360.](https://github.com/JuliaBinaryWrappers/CUDA_loader_jll.jl/commit/ecb51de22e7681fc18bf6e7b58f5df0c05f9d9eb#diff-1de81b882b140a45cfbcedd944f143a37759b544a9efc0f7008302578afa6360)